controller/volume_controller.go

```diff
@@ -3022,6 +3022,159 @@ func (vc *VolumeController) ResolveRefAndEnqueue(namespace string, ref *metav1.O
 	vc.enqueueVolume(volume)
 }
 
+// generateJobName returns a name that contains prefix and 16 random characters
+func generateJobName(prefix string) string {
```

Comment:
This function is just used by `generateRecurringJobName`, so suggest merge into `generateRecurringJobName`

Reply:
Merged into the caller function `generateRecurringJobName`.